replicated-security/duality-second-rehearsal-1/README.md

```diff
@@ -0,0 +1,114 @@
+
+* **Chain-ID**: `duality-second-rehearsal-1`
+* **minimum-gas-prices**: ``
+* **timeout_commit**: `2s`
+* **Spawn time**: `2023-07-05T15:00:00Z`
+* **GitHub repo**: [duality-labs/duality](https://github.com/duality-labs/duality.git)
+* **Release**: [`v0.3.0`](https://github.com/duality-labs/duality/releases/tag/v0.3.0)
+<!-- * **Genesis file with CCV state:** [duality-second-rehearsal-1-genesis.json](duality-second-rehearsal-1-genesis.json) -->
+
+* **Reference binary**: [dualityd-linux-amd64](./dualityd-linux-amd64.tar.gz)
+* **Binary sha256sum**: `d42f09d0af91b921f4a1e4635e55270fe0a7a3383ec185e2bee92bdc7cf6df82`
+* **Genesis file _without CCV state_:** [duality-second-rehearsal-1-genesis-without-ccv.json](duality-second-rehearsal-1-genesis-without-ccv.json), verify with `shasum -a 256 duality-second-rehearsal-1-genesis-without-ccv.json`
+* **SHA256 for genesis file _without CCV state_**: `e095ed2c8226b88d73776f268b1bdfe74cd455a0fc2a598f26035e64fde39d2d`
+
+* Genesis file hash
+  * The SHA256 is used to verify against the genesis file (without CCV state) that the proposer has made available for review.
+  * The `duality-second-rehearsal-1-genesis-without-ccv.json` file cannot be used to run the chain: it must be updated with the CCV (Cross Chain Validation) state after the spawn time is reached.
+  * The genesis file includes funds for a relayer and a faucet account as well as account with funds for different internal needs, `signed_blocks_window` has been set to `864000`, and `min_signed_per_window` has been set to `5%`.
+* Binary hash
+  * The `dualityd-linux-amd64` binary is only provided to verify the SHA256. It was built with Interchain Security release [`v3.0.0-rc2`](https://github.com/cosmos/interchain-security/releases/tag/v3.0.0-rc2). You can generate the binary following the build instructions in that repo or using one of the scripts provided here.
+* Spawn time
+  * Even if a proposal passes, the CCV state will not be available from the provider chain until after the spawn time is reached.
+
+> If you want to build genesis by your own please take into account difference between the `ccvconsumer` section generated by version v9.0.2 of `gaiad` and the `ccvconsumer` section required for duality to run. Please check and add `"soft_opt_out_threshold": "0.05"` to the `params` section if it absent.
+
+For more information regarding the consumer chain creation process, see [CCV: Overview and Basic Concepts](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md).
+
+## Endpoints
+
+Endpoints are exposed as subdomains for the sentry and snapshot nodes (described below) as follows:
+
+* `https://rpc.testnet-2.duality.xyz`
+* `https://grpc.testnet-2.duality.xyz`
+* `https://rest.testnet-2.duality.xyz` (or `https://lcd.testnet-2.duality.xyz`)
+
+Seed nodes:
+
+1. `cebd80328abca02408d1b7199f9b1a835a14aa31@p2p.testnet-1.duality.xyz:26656`
```

Comment:
https://github.com/cosmos/testnets/pull/454/files updated the seed node to `75546756a3d3189594ebce101700605c1e956053@p2p.testnet-1.duality.xyz:26656` - did you want that change to be reflected here too?